### DISCUSSION TITLE
doubt in evaluator used by gluonts

### DISCUSSION THREAD

**Discussion by pratikgehlott** [awslabs/gluonts#1636] [Q&A]
Hello,

So, I was using this evaluator for my project and came across this function that says (attached in the below image),
I need to know why the evaluation function is evaluating against the forecasts generated, (because according to me, we don't have any ground truth value for forecasts, so comparing against it won't make any sense.) Can anyone explain me this?

![image](https://user-images.githubusercontent.com/65299118/126449155-1392d7b5-d9e7-4d43-85a4-71bf85bf2ffa.png)

**Comment by pratikgehlott** [comment]
@mbohlkeschneider, @lostella can you guys explain me this?

**Comment by timoschowski** [comment]
Hi @Pratik325 , to be able to evaluate the quality of your forecasts quantitatively, you need to compare against observations or actual values. The evaluation functions that we have use the observations/actuals and compares them against forecasts to compute some error measure on them, sMAPE, MASE, CRPS, something else. So, this will only of course work in a backtesting scenario where you have ground truth (past observed values) available.

**Comment by mbohlkeschneider** [comment]
Has this been clarified @Pratik325 ?
